pycaret/containers/models/time_series.py

```diff
@@ -271,6 +276,11 @@ def __init__(self, globals_dict: dict) -> None:
         )
 
 
+######################################
+#### CLASSICAL STATISTICAL MODELS ####
+######################################
+
+
```

Comment:
To make more clear and clean the `__init__` of the ArimaContainer, is it possible to decompose this process into methods that get called in the `__init__` ?. For example `return_order_related_params` can be a method of this class called during init 

```python
class ArimaContainer(TimeSeriesContainer):

    def __init__():
          ....
         self.return_order_related_params()

    def return_order_related_params(self):

```